### ISSUE TITLE
Quick-search for certain brenda ontology terms returns a blank page

### ISSUE THREAD

**Issue by cmdcolin** [elsiklab/bovinemine#2]
It seems to look for chromosomeLocDisplayer.jsp which crashes the jsp

```
521645961 [http-bio-8080-exec-3852] ERROR org.apache.struts.tiles.taglib.InsertTag  - ServletException in '/keywordSearchResults.jsp': javax.servlet.jsp.JspException: ServletException in '/model/chromosomeLocDisplayer.jsp': Error reading 'locatedOn' on type org.intermine.model.bio.LocationShadow
    org.apache.jasper.JasperException: javax.servlet.ServletException: javax.servlet.jsp.JspException: ServletException in '/model/chromosomeLocDisplayer.jsp': Error reading 'locatedOn' on type org.intermine.model.bio.LocationShadow
    at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:549)
    at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:455)
    at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:395)
    at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:339)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:727)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
    at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:748)
```

Also just wanted to start keeping track of things on github :)

**Comment by cmdcolin** [comment]
Appears to be fixed! Search BTO:0000004 as example

**Comment by deepakunni3** [comment]
This is a common bug. Its not there always and I am not sure why that is.
